Source/ToolCore/Build/BuildAndroid.cpp

```diff
@@ -260,6 +261,25 @@ void BuildAndroid::RunAntDebug()
     args.Push("\"" + antPath + "\"");
     args.Push("debug");
 #endif
+#ifdef ATOMIC_PLATFORM_LINUX 
+
+    String antCommand = tprefs->GetAntPath();
+    if ( antCommand.Empty() ) // user didnt fill it out, use installed one
+    {
+ 		antCommand = "/usr/bin/ant"; // system default if installed
+	}
+	else
+	{
+		antCommand.Append("/ant"); 
+	}
```

Comment:
I hadn’t noticed the indentation differences even when I went thru the diffs. I will fix my editors to do the right thing.